services/network_config_manager/src/main/java/com/futurewei/alcor/netwconfigmanager/cache/VpcResourceCache.java

```diff
@@ -28,7 +27,7 @@
 @ComponentScan(value = "com.futurewei.alcor.common.db")
 public class VpcResourceCache {
 
-    // Map <VNI, Map<PIP, List<ResoruceIDType>>
+    // Map <VNI, Map<PIP, ResourceMetadata>
```

Comment:
TXN:
Need a "CacheFactory    cacheFactory" member which will be assigned the cacheFactory param passed into the Constructor.

The following method is also needed:
public Transaction getTransaction() {
        return cacheFactory.getTransaction();
}

The same change is needed in HostResourceMetadataCache and, ResourceCacheManager classes.